clarifai/cli/compute_cluster.py

```diff
@@ -0,0 +1,51 @@
+import click
+from clarifai.cli.base import cli
+from clarifai.client.user import User
+from clarifai.utils.cli import display_co_resources
+
+
+@cli.group(['computecluster', 'cc'])
+def computecluster():
+  """Manage Compute Clusters: create, delete, list"""
+  pass
+
+
+@computecluster.command()
+@click.option(
+    '--config',
+    type=click.Path(exists=True),
+    required=True,
+    help='Path to the compute cluster config file.')
+@click.option(
+    '-cc_id',
+    '--compute_cluster_id',
+    required=False,
+    help='New Compute Cluster ID for the compute cluster to create.')
+@click.pass_context
+def create(ctx, config, compute_cluster_id):
+  """Create a new Compute Cluster with the given config file."""
+  user = User(user_id=ctx.obj['user_id'], pat=ctx.obj['pat'], base_url=ctx.obj['base_url'])
+  if compute_cluster_id:
+    user.create_compute_cluster(config, compute_cluster_id=compute_cluster_id)
```

Comment:
I believe this will overwrite the cc_id if its present in config

Reply:
Yes